cristin-import/src/main/java/no/unit/nva/cristin/mapper/CristinMapper.java

```diff
@@ -527,21 +527,13 @@ private String extractAbstract() {
     }
 
     private List<String> extractTags() {
-        if (extractCristinTags() == null) {
+        var tags = extractCristinTags();
+        if (isNull(tags)) {
             return null;
         }
-        List<String> listOfTags = new ArrayList<>();
-        for (CristinTags cristinTags : extractCristinTags()) {
-            if (cristinTags.getBokmal() != null) {
-                listOfTags.add(cristinTags.getBokmal());
-            }
-            if (cristinTags.getEnglish() != null) {
-                listOfTags.add(cristinTags.getEnglish());
-            }
-            if (cristinTags.getNynorsk() != null) {
-                listOfTags.add(cristinTags.getNynorsk());
-            }
-        }
-        return listOfTags;
+        return tags.stream()
+                   .flatMap(tag -> Stream.of(tag.getBokmal(), tag.getEnglish(), tag.getNynorsk()))
+                   .filter(Objects::nonNull)
```

Comment:
Maybe also trim the strings before checking for unique strings?

Reply:
Adding following:

```
tags.stream()
                   .flatMap(tag -> Stream.of(tag.getBokmal(), tag.getEnglish(), tag.getNynorsk()))
                   .filter(Objects::nonNull)
                   .collect(Collectors.toMap(String::trim, Function.identity(), (a, b) -> a, LinkedHashMap::new))
                   .values()
                   .stream()
                   .toList();
```

It trims string before comparison, removes duplicates, but return original values.